woocommerce-gateway-ebanx/assets/js/disable-ebanx-gateways.js

```diff
@@ -0,0 +1,30 @@
+;(function($){
```

Comment:
Please, take care with this code, because it was purposely added to be fast and avoid rendering problems when page loads.

Do tests with slow broadband.